--- yt_dlp/update.py.orig	2021-10-09 23:03:01.000000000 -0500
+++ yt_dlp/update.py	2021-10-12 23:55:02.000000000 -0500
@@ -51,7 +51,7 @@
     'dir': 'Auto-update is not supported for unpackaged windows executable. Re-download the latest release',
     'py2exe': 'There is no official release for py2exe executable. Build it again with the latest source code',
     'source': 'You cannot update when running from source code',
-    'unknown': 'It looks like you installed yt-dlp with a package manager, pip, setup.py or a tarball. Use that to update',
+    'unknown': 'It looks like you installed yt-dlp with MacPorts. Please use "sudo port selfupdate && sudo port upgrade yt-dlp" to update.',
 }
 
 
